src/xenia/kernel/xboxkrnl/xboxkrnl_io.cc

```diff
@@ -265,6 +265,90 @@ dword_result_t NtReadFile(dword_t file_handle, dword_t event_handle,
 }
 DECLARE_XBOXKRNL_EXPORT2(NtReadFile, kFileSystem, kImplemented, kHighFrequency);
 
+dword_result_t NtReadFileScatter(dword_t file_handle, dword_t event_handle,
+                                 lpvoid_t apc_routine_ptr, lpvoid_t apc_context,
+                                 pointer_t<X_IO_STATUS_BLOCK> io_status_block,
+                                 lpdword_t buffer_ptr, dword_t length,
```

Comment:
`segment_array` would be easier to understand in the future, since "buffer" in file reading usually implies just one area where to read the data to.

Reply:
Agreed, will change that now